src/js-display-panel.ts

```diff
@@ -0,0 +1,135 @@
+import { ToggleStatus } from './js-feature-toggle-manager';
+import { QueryStringReader } from './github/gtmsportswear/js-utilities@1.0.0/js-utilities';
+
+export abstract class DisplayPanel {
+  public draw(): Element {
+    const panel = document.createElement('div');
+    panel.classList.add('display-panel');
+
+    return panel;
+  }
+}
+
+export class FeatureToggleDisplayPanel extends DisplayPanel {
+  constructor(private toggles: ToggleStatus[]) {
```

Comment:
I don't understand what is going on with this constructor. It seams to be calling the parents class constructor which is just a default constructor. Why is it then being passed in the toggles? Are they being used somehow?